notebooks/examples/worldtree_100_dataset.json

```diff
@@ -280,10 +280,10 @@
                 "type": "multiplechoice",
                 "cot_type": "list",
                 "choices": [
-                    "1 liter of ice at a temperature of -50\u00c2\u00b0C",
-                    "10 liters of ice at a temperature of -75\u00c2\u00b0C",
-                    "1 liter of water at a temperature of 75\u00c2\u00b0C",
-                    "10 liters of water at a temperature of 50\u00c2\u00b0C"
+                    "1 liter of ice at a temperature of -50\u00b0C",
+                    "10 liters of ice at a temperature of -75\u00b0C",
+                    "1 liter of water at a temperature of 75\u00b0C",
+                    "10 liters of water at a temperature of 50\u00b0C"
                 ],
```

Comment:
It seems we have encoding problems when dumping the json. This is not your fault, I'll investigate this.